### ISSUE TITLE
Testing(Scripts/e2e)

### ISSUE THREAD

**Issue by crcommons** [ushahidi/platform#2576]
Investigate which libraries we could use for unit/e2e-testing
Investigate how we can test for accessibility (Robbie had a good idea about adding test-scripts that ensure we are not cheating, for example find all img-elements and check for alt-texts, fail if no alt-text is included)

**Comment by crcommons** [comment]
@Angamanga @rowasc Tagging you here solely because you had originally signed up for this one and want to keep a record. You don't need to reassign yourself.

**Comment by willdoran** [comment]
I would strongly recommend that this section focuses heavily on how this will be QA'd by a human primarily because we are a good distance from real automation which is like Data Model a legit project of its own.

If however there is an immediate way we can do automation cool but please keep in mind that it is likely to be humans that have to run these scripts.
